src/main/scala/net/ruippeixotog/scalascraper/util/ProxyUtils.scala

```diff
@@ -0,0 +1,32 @@
+package net.ruippeixotog.scalascraper.util
+
+object ProxyUtils {
+
+  /**
+   * Set HTTP and HTTPS proxy JVM-wide
+   */
+  def setProxy(host: String, port: Int): Unit = setProxy(Some(host), Some(port))
+
+  /**
+   * Remove HTTP and HTTPS proxy configuration JVM-wide
+   */
+  def removeProxy(): Unit = setProxy(None, None)
+
+  /**
+   * Implicit conversions to String or Java null
+   */
+  private implicit def optionStrToJavaStr(value: Option[String]): String = value.getOrElse(null)
```

Comment:
I think there is no need for such extra implicit conversions here, as the code for converting `Option[Int]` and `Option[String]` to `String` can be very concise inlined. For example, you can implement `setProxy` as:

```
System.setProperty("http.proxyHost", host.orNull)
System.setProperty("http.proxyPort", port.map(_.toString).orNull)
System.setProperty("https.proxyHost", host.orNull)
System.setProperty("https.proxyPort", port.map(_.toString).orNull)
```

That would remove the need for the implicits and would still provide readable code IMO.